# Execute the position goal for given planning group

# Requests
# group_name: The group providing the reference frame
# goal_type: BASE_ABS: pose wrt the global base frame (robot base)
#            BASE_REL: pose wrt the base frame transferred to current eef base position
#            EEF: pose wrt the eef frame
# goal: Position of the goal
# tolerance [opt]: Tolerance for considering the goal has been reached, default 0.01 m/rad

std_msgs/Header header
string group_name
uint8 BASE_ABS=0
uint8 BASE_REL=1
uint8 EEF=2
uint8 goal_type
geometry_msgs/Point goal
float64 tolerance
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status